frontend/packages/rhoas-plugin/src/topology/components/TopologyKafkaPanel.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
+import { Node } from '@patternfly/react-topology';
+import * as UIActions from '@console/internal/actions/ui';
+import './TopologyKafkaPanel.css';
+import { navFactory, SimpleTabNav } from '@console/internal/components/utils';
+import { ResourcesComponent } from './ResourceComponent';
+import { DetailsComponent } from './DetailsComponent';
+
+type PropsFromState = {
+  selectedDetailsTab?: any;
```

Comment:
Isn't this of type `string` ? or `string` ?
We don't want to use the `any` type.